torch/quantization/fx/_equalize.py

```diff
@@ -220,3 +222,215 @@ def node_supports_equalization(node: Node, modules) -> bool:
     elif node.op == 'call_function':
         return node.target == nn.functional.linear
     return False
+
+def get_weight_eq_obs(node: Node, model: GraphModule, modules: Dict[str, nn.Module]):
```

Comment:
what is `node` here?  Does it correspond to the input equalization observer?  If yes, might be good to name it something like `input_eq_obs_node`.